.env.dev

```diff
@@ -0,0 +1,14 @@
+# Hare needs this
+HOST=0.0.0.0
+
+SHOW_EXAMPLES=true
+
+# Where to write hare-access.log, hare-error.log
+#LOG_DIR=/foo/bar/hare/logs
+
+# Fine-tune which facilities you want to STDOUT #TODO
+# NOTE: Work needs to be done so hare logs could go there too.
+#       And we should review use of cross-env to use env-cmd
+#      https://github.com/kentcdodds/cross-env#other-solutions
+#DEBUG=nuxt:*,axios:*,koa:*,koa-session:*,koa-router:*,follow-redirects:*
```

Comment:
This is so cool!

Maybe its better to use Environment Variables instead of loading a long string in packages.json

Reply:
HOST is needed inside the Docker image, I should say.